logs.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: Zookeeper Provisioning
+  hosts: zookeeper
+  gather_facts: false
+  tags: zookeeper
+  tasks:
+    - include_role:
+        name: confluent.common
+        tasks_from: failure_handling.yml
```

Comment:
Also wondering about renaming the task file, since you are using it just for log exporting and not failure_handling... not too important.

I think we also need to pass these new variables (user, group) within the health_check.yml files in each component.

Reply:
Agree. Renaming file name and directory.